docs/developing/ui-component-testing.md

```diff
@@ -0,0 +1,32 @@
+# UI Component Testing
+
+This document gives you a process to follow to check that any UI components
+you have created will be in tip-top shape for the site.
+
+## Browsers to check
+
+It doesn't have to look exactly the same in every browser, but it should look similar and work properly in current versions of these evergreen browsers:
+
+### Desktop
+
+* Firefox
+* Opera
+* Chrome
+* Edge
+* IE 11
+* Safari (latest and latest but one)
+
+### Mobile
+
+Latest versions of:
+
+* iPhone: Chrome and Safari
+* iPad: Chrome and Safari
```

Comment:
Is it worth checking Chrome on iOS considering it's just a WebKit view like Safari?

Reply:
I've observed differences between Chrome and Safari on iOS, even since Chrome changed to use the Webkit view. Main areas are around differing UI responsiveness, and how length calculation assumptions can differ when full screen is involved.